### PR TITLE
Comments now in state managed by stream post. Saved when modal closes.

### PR DESCRIPTION
**Pull request by shchau**
Comments now in state managed by stream posts. So it's saved when modal closes.

### CODE REVIEW

File: frontend/src/components/CommentsOnPost.jsx [hunk 1]
```diff
@@ -44,6 +49,7 @@ class CommentsOnPost extends Component {
 							comments: commentList,
 							isFetching: false,
 						});
+						this.updateComments(this.state.comments);
```
Comment (hiufungk) [inline frontend/src/components/CommentsOnPost.jsx:52]: why not this.props.updateComments(newComments); directly lol

File: frontend/src/components/StreamPost.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ class StreamPost extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
+			comments: this.props.comments,
```
Comment (hiufungk) [inline frontend/src/components/StreamPost.jsx:25]: prop types？

File: frontend/src/components/CommentsOnPost.jsx (hunk 1 above)
Comment (shchau) [reply frontend/src/components/CommentsOnPost.jsx:52]: Just in case I want updateComments to do other things

Comment (hiufungk) [reply frontend/src/components/CommentsOnPost.jsx:52]: sure

**Review by htruong4 — APPROVED** [review]

**Review by hiufungk — APPROVED** [review]

**Pull request merged**